source/common/http/codec_client.h

```diff
@@ -183,8 +183,10 @@ class CodecClient : protected Logger::Loggable<Logger::Id::client>,
   }
 
   void onIdleTimeout() {
-    host_->cluster().trafficStats()->upstream_cx_idle_timeout_.inc();
-    close();
```

Comment:
what's happening if we close a connection which is not established? I wonder what is happening in such case. I believe the close is just ignored